### PR TITLE
chore(issuer): Enable the batchIssue() method in ERC1888

### PR DESCRIPTION
**Pull request by josipbagaric**
- [x] Add support for Batch issuing
- [x] Only generate the validity function on-chain
- [x] Update ERC-1888 to the latest spec on https://github.com/ethereum/EIPs/issues/1888
- [x] Move batch actions into a separate facade

### CODE REVIEW

File: packages/traceability/issuer/contracts/Registry.sol [hunk 1]
```diff
@@ -13,20 +13,86 @@ contract Registry is ERC1155, ERC1888 {
...
+		require(_values.length == _validityCalls.length, "Registry::batchIssueMultiple: _values and _validityCalls arrays have to be the same length");
+
+		uint256[] memory _ids = new uint256[](_values.length);
+
+		address operator = _msgSender();
+
+		for (uint256 i = 0; i < _values.length; i++) {
+			_ids[i] = i + _latestCertificateId + 1;
+			_validate(operator, _validityCalls[i]);
+		}
```
Comment (kosecki123) [inline packages/traceability/issuer/contracts/Registry.sol]: This potentially can be a separate function, same code used in `batchIssue`

**Comment by josipbagaric** [comment]
@kosecki123 added some more comments to the smart contracts in the meantime. Now ready for a review again.

File: docs/traceability.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Traceability SDK
```
Comment (JosephBagaric) [inline docs/traceability.md:1]: Auto-generated docs.

File: docs/traceability/contracts/ERC1888/ERC1888.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+## `ERC1888`
```
Comment (josipbagaric) [inline docs/traceability/contracts/ERC1888/ERC1888.md:1]: Auto-generated docs.

File: packages/traceability/issuer/src/blockchain-facade/CertificateBatchOperations.ts [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        new Certificate(certId, blockchainProperties).sync()
+    );
+
+    const { registry, activeUser } = blockchainProperties;
+    const registryWithSigner = registry.connect(activeUser);
+
+    const activeUserAddress = await activeUser.getAddress();
+    const fromAddress = from ?? activeUserAddress;
+
+    const certificates = await Promise.all(certificatesPromises);
```
Comment (kosecki123) [inline packages/traceability/issuer/src/blockchain-facade/CertificateBatchOperations.ts:69]: Nice touch, doing other stuff while waiting for certificates to sync 👍

**Review by kosecki123 — APPROVED** [review]

**Pull request merged**